components/scream/cime_config/buildnml

```diff
@@ -690,47 +697,6 @@ def create_input_files(caseroot, screamroot, rundir):
 
     return result
 
-###############################################################################
```

Comment:
This change is unrelated. I just felt that buildnml should be the first function you see as you scroll up from _main_func since buildnml is the "main" function.